cms/release_calendar/models.py

```diff
@@ -174,6 +177,20 @@ def get_context(self, request: "HttpRequest", *args: Any, **kwargs: Any) -> dict
         context["table_of_contents"] = self.table_of_contents
         return context
 
+    @property
+    def release_date_value(self) -> str:
+        if self.release_date_text and self.status == ReleaseStatus.PROVISIONAL:
+            return self.release_date_text
+        return ons_date_format(self.release_date, "DATETIME_FORMAT")
+
+    @property
+    def next_release_date_value(self) -> str | None:
+        if self.next_release_date:
+            return ons_date_format(self.next_release_date, "DATETIME_FORMAT")
+        if self.next_release_date_text:
+            return self.next_release_date_text
+        return None
```

Comment:
The help text for `next_release_date_text` says "Override next release date". So this should be flipped (it was correct before 080b253

Reply:
saying that, we do have validation in the form that prevents both to be added. so let's leave as is.